website/pages/docs/configuration/telemetry.mdx

```diff
@@ -37,6 +37,22 @@ The following options are available on all telemetry configurations.
 - `enable_hostname_label` `(bool: false)` - Specifies if all metric values should
   contain the `host` label with the local hostname. It is recommended to enable
   `disable_hostname` if this option is used.
+- `lease_metrics_epsilon` `(string: "1hr")` - Specifies the time difference between collections
+  of leases by expiry time. For example, for the default value of 1 hour, the `vault.expire.leases.by_expiration`
+  metric will aggregate the total number of expiring leases for 1 hour buckets, starting from the current time.
+  Note that leases are put into buckets by rounding. For example, if `lease_metrics_epsilon` is set to 1hr and 
+  lease A expires 25 minutes from now, and lease B expires 35 minutes from now, then lease A will be in a
+  different bucket from lease B. 
```

Comment:
I did not notice this, and it's somewhat surprising to me.  Maybe you could be more explicit in your example and say something like:

For the default 1 hour buckets, the first bucket consists of leases expiring between 0 and 29 minutes in the future, the second bucket consists of leases expiring between 30 and 99 minutes in the future, etc.

Rather than just stating that they are "different buckets".

(Alternatively, it's not too late to reconsider and round down so that bucket 0 is "0-59 minutes", bucket 1 is "60-119 minutes", etc.)

Reply:
I'm happy to do floors instead of rounds (so 0-59 --> bucket 0, 60-119 --> bucket 1, etc.), but I thought rounding would be more descriptive. For example, if 100 leases expire at minute 55 and 100 at minute 65, I as a customer would probably want the metric to report 200 in bucket 1, and 0 in bucket 0. But I can see it going either way. 

Definitely, can rewrite the example to be more explicit!